runtimes/rust/Cargo.toml

```diff
@@ -34,6 +34,7 @@ bincode = "1.2.1"
 criterion = "0.3.0"
 pretty_assertions = "0.6.1"
 rand = "0.7"
+semver = { version = "0.9.0", features = ["serde"] }
```

Comment:
Nit: `exonum` reexports `Version` in `runtime::versioning` module, so you may avoid specifying this dependency.